includes/ui/class-ui.php

```diff
@@ -33,12 +33,20 @@ class UI {
 	 *
 	 * @var bool
 	 */
-	private $has_ui = false;
+	private $has_ui;
+
+	/**
+	 * An array of css variables to be enqueued with the styles.
+	 *
+	 * @var array
+	 */
+	private $css_variables;
 
 	/**
 	 * Instance constructor
 	 */
 	private function __construct() {
+		$this->has_ui = false;
```

Comment:
```suggestion
		$this->has_ui        = false;
		$this->css_variables = [];
```

Just to initialize everything on constructor (so I think we can avoid some potential notices/warnings here)